common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -457,7 +460,11 @@ workflow_state = ? ` +
 		`and task_id = ? `
 
 	templateUpdateWorkflowExecutionWithReplicationQuery = `UPDATE executions ` +
```

Comment:
plz see above

Reply:
before actual update, we should do an assertion that at most one (replication state / version history) is set (for global domain workflow)

for local domain workflow, let us set the next event ID to empty event ID, current branch token to be empty.

Reply:
Done